tests/test_doc.rs

```diff
@@ -0,0 +1,11 @@
+extern crate docmatic;
+
+#[test]
+fn test_readme() {
```

Comment:
In theory, running the lib.rs docs as tests verifies the same behavior as these tests